pre_commit_hooks/debug_statement_hook.py

```diff
@@ -42,6 +42,12 @@ def check_file_for_debug_statements(filename):
         print('\t' + traceback.format_exc().replace('\n', '\n\t'))
         print()
         return 1
+    except Exception:
+        print('{} - Caught exception while reading file'.format(filename))
+        print()
+        print('\t' + traceback.format_exc().replace('\n', '\n\t'))
+        print()
+        return 1
```

Comment:
I think you'll find that this is already fixed: https://github.com/pre-commit/pre-commit-hooks/pull/230